js/layout-javascript/news-feed-code.js

```diff
@@ -165,6 +167,40 @@ DynamicList.prototype.attachObservers = function() {
         console.error(error);
       });
     })
+    .on('click', '.sort-group .list-sort li', function (e) {
+      e.stopPropagation();
+
+      var $sortListItem = $(e.currentTarget);
+      var oldSortOrder = $sortListItem.data('sortOrder');
+      var $sortOrderIcon = $sortListItem.find('i');
+      var $sortList = _this.$container.find('.list-sort li');
+      var sortClasses = {
+        'none': 'fa-sort',
+        'asc': 'fa-sort-asc',
+        'desc': 'fa-sort-desc'
+      };
+      var newSortOrder = {
+        'none': 'asc',
+        'asc': 'desc',
+        'desc': 'asc'
+      };
+
+      _this.sortOrder = newSortOrder[oldSortOrder];
```

Comment:
Don't need the `newSortOrder` or `oldSortOrder` variables.

```suggestion
      _this.sortOrder = $sortListItem.data('sortOrder') === 'asc' ? 'desc' : 'asc';
```